tests/pyfunc/test_dependencies_functions.py

```diff
@@ -110,3 +111,81 @@ def mock_get_installed_version_fn(package, module=None):
                     "detecting model dependency mismatches"
                 )
             )
+
+
+@pytest.mark.large
+def test_get_model_dependencies(tmpdir):
+    with open(os.path.join(tmpdir.strpath, "MLmodel"), "w") as f:
+        f.write(
+            """
+artifact_path: model
+flavors:
+  python_function:
+    env: conda.yaml
+    loader_module: mlflow.sklearn
+    model_path: model.pkl
+    python_version: 3.7.12
+model_uuid: 722a374a432f48f09ee85da92df13bca
+run_id: 765e66a5ba404650be51cb02cda66f35"""
+        )
+
+    req_file_path = os.path.join(tmpdir.strpath, "requirements.txt")
+    with open(req_file_path, "w") as f:
+        f.write(
+            """
+mlflow
+cloudpickle==2.0.0
+scikit-learn==1.0.2"""
+        )
+
+    conda_yml_path = os.path.join(tmpdir.strpath, "conda.yaml")
+    with open(conda_yml_path, "w") as f:
+        f.write(
+            """
+channels:
+- conda-forge
+dependencies:
+- python=3.7.12
+- pip=22.0.3
+- scikit-learn=0.22.0
+- tensorflow=2.0.0
+- pip:
+  - mlflow
+  - cloudpickle==2.0.0
+  - scikit-learn==1.0.1
+name: mlflow-env"""
+        )
+
+    model_path = tmpdir.strpath
+
+    # Test getting pip dependencies
+    assert get_model_dependencies(model_path, format="pip") == req_file_path
+
+    # Test getting pip dependencies will print instructions on databricks
+    with mock.patch("mlflow.pyfunc._logger.info") as mock_log_info:
+        get_model_dependencies(model_path, format="pip")
+        mock_log_info.assert_not_called()
+
+        mock_log_info.reset_mock()
+        with mock.patch("mlflow.pyfunc.is_in_databricks_runtime", lambda: True):
```

Comment:
```suggestion
        with mock.patch("mlflow.pyfunc.is_in_databricks_runtime", return_value=True):
```

Can we use `return_value` ([reason](https://github.com/mlflow/mlflow/pull/5386#discussion_r808718952))?